app/_data/docs_nav_kic_2.10.x.yml

```diff
@@ -4,163 +4,229 @@ generate: true
 items:
   - title: Introduction
     icon: /assets/images/icons/documentation/icn-kubernetes-color.svg
-    url: /kubernetes-ingress-controller/
+    url: /kubernetes-ingress-controller/2.10.x/
     absolute_url: true
     items:
       - text: FAQ
         url: /faq
+        src: /kic-v2/faq
       - text: Version Support Policy
         url: /support-policy
+        src: /kic-v2/support-policy
       - text: Stages of Software Availability
         url: /availability-stages
+        src: /kic-v2/availability-stages
       - text: Changelog
-        url: https://github.com/Kong/kubernetes-ingress-controller/blob/main/CHANGELOG.md
+        url: >-
```

Comment:
Why does this get added?

Reply:
The YAML parser thinks that the URL is too long for a single line

Reply:
I guess it doesn't seem to hurt anything, the output works fine.

Reply:
Yep, it's valid YAML
![CleanShot 2023-10-06 at 17 28 35](https://github.com/Kong/docs.konghq.com/assets/59130/368ac314-db16-4e52-b5aa-9160c012c98f)